test/mpi/mpiexec.jl

```diff
@@ -10,6 +10,8 @@ function run_mpi_driver(;procs,file)
     else
       run(`$cmd -n $procs $(Base.julia_cmd()) --project=$repodir $(joinpath(mpidir,file))`)
     end
-    @test true
+    # This line will be reached if and only if the command launched by `run` runs without errors.
+    # Then, if we arrive here, the test has succeeded.
+    @test true  
```

Comment:
Just FYI, there is the `success` function (https://docs.julialang.org/en/v1/base/base/#Base.success) for just making sure that a command returns 0, so the `run` calls above could be replaced with e.g.
```
@test success(`$cmd -n ...`)
```

Reply:
Very good point!